INFO  2014-03-29 14:17:21,047 (Saros.java:256) Starting Saros 13.12.6 running:
  Java Version: 1.8.0
  Java Vendor: Oracle Corporation
  Eclipse Runtime Version: 3.9.100.v20131218-1515
  Operating System: Windows 8.1 (6.3)
  Hardware Architecture: amd64
INFO  2014-03-29 14:17:21,184 (XMPPAccountStore.java:103) loaded 1 accounts
INFO  2014-03-29 14:17:21,269 (PreferenceInitializer.java:53) initializing preference default values
DEBUG 2014-03-29 14:17:21,291 (DataTransferManager.java:444) used transport order for the current XMPP connection: [de.fu_berlin.inf.dpp.net.internal.Socks5Transport@507d20bb, de.fu_berlin.inf.dpp.net.internal.IBBTransport@9fec931]
DEBUG 2014-03-29 14:17:21,425 (StreamServiceManager.java:225) StreamServices added:
  SendFile
  AudioService
  VideoSharing
DEBUG 2014-03-29 14:17:21,567 (SarosAnnotation.java:112) resetting annotation colors
WARN  2014-03-29 14:17:22,380 (NullSarosSession.java:64) unexpected method call
WARN  2014-03-29 14:17:22,395 (NullSarosSession.java:64) unexpected method call
DEBUG 2014-03-29 14:17:28,831 (SharedDocumentProvider.java:79) SharedDocumentProvider created by Eclipse
WARN  2014-03-29 14:17:34,702 (NullSarosSession.java:64) unexpected method call
WARN  2014-03-29 14:17:34,718 (NullSarosSession.java:64) unexpected method call
DEBUG 2014-03-29 14:17:36,499 (WhiteboardManager.java:108) WhiteboardManager instantiated
DEBUG 2014-03-29 14:17:36,515 (DocumentRecord.java:93) Added Node: attr(139609905651574866) name=svg
DEBUG 2014-03-29 14:17:36,921 (SarosNet.java:423) started Socks5 proxy on port: 7777 [listening on all interfaces]
DEBUG 2014-03-29 14:17:37,536 (SarosNet.java:433) found interface addresses: [192.168.56.1, 192.168.178.42, fe80:0:0:0:6916:72e8:fdbf:3bcd%wlan0, fe80:0:0:0:6162:f90:6766:5971%eth5]
DEBUG 2014-03-29 14:17:37,598 (SarosNet.java:383)  new connection state == CONNECTING
DEBUG 2014-03-29 14:17:39,617 (StunServiceImpl.java:173) discovered public WAN-IP: 2.241.41.67 through interface 192.168.178.42
DEBUG 2014-03-29 14:17:40,033 (SarosNet.java:383) pascal.borchert.games@googlemail.com/SarosDA175816 new connection state == CONNECTED
DEBUG 2014-03-29 14:17:40,033 (DataTransferManager.java:444) used transport order for the current XMPP connection: [de.fu_berlin.inf.dpp.net.internal.Socks5Transport@507d20bb, de.fu_berlin.inf.dpp.net.internal.IBBTransport@9fec931]
DEBUG 2014-03-29 14:17:40,033 (SkypeManager.java:286) Refreshing Skype username cache...
WARN  2014-03-29 14:17:48,941 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
ERROR 2014-03-29 16:50:55,742 (SarosNet.java:95) XMPP connection error: 
java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
ERROR 2014-03-29 16:50:55,948 (SarosNet.java:385)  new connection state == ERROR
java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
DEBUG 2014-03-29 16:50:55,996 (SarosNet.java:383)  new connection state == NOT_CONNECTED
INFO  2014-03-30 13:43:58,327 (XMPPAccountStore.java:161) saved 1 accounts
DEBUG 2014-03-30 13:43:58,374 (SarosNet.java:423) started Socks5 proxy on port: 7777 [listening on all interfaces]
DEBUG 2014-03-30 13:43:58,765 (SarosNet.java:433) found interface addresses: [192.168.56.1, 192.168.178.42, fe80:0:0:0:6916:72e8:fdbf:3bcd%wlan0, fe80:0:0:0:6162:f90:6766:5971%eth5]
DEBUG 2014-03-30 13:43:58,765 (SarosNet.java:383)  new connection state == CONNECTING
DEBUG 2014-03-30 13:43:59,405 (SarosNet.java:383) pascal.borchert.games@googlemail.com/SarosDFE9872D new connection state == CONNECTED
DEBUG 2014-03-30 13:43:59,405 (DataTransferManager.java:444) used transport order for the current XMPP connection: [de.fu_berlin.inf.dpp.net.internal.Socks5Transport@507d20bb, de.fu_berlin.inf.dpp.net.internal.IBBTransport@9fec931]
DEBUG 2014-03-30 13:43:59,405 (SkypeManager.java:286) Refreshing Skype username cache...
DEBUG 2014-03-30 13:43:59,421 (StunServiceImpl.java:173) discovered public WAN-IP: 2.241.41.67 through interface 192.168.178.42
WARN  2014-03-30 13:44:09,296 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-03-30 13:44:30,734 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-30 13:45:00,750 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-30 14:05:15,602 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-30 14:05:45,612 (DiscoveryManager.java:445) Service Discovery failed on recipient gabrielwilde@gmail.com/MessagingAe01cc205 server: xmpp.l.google.com
No response from the server.: 
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:554)
	at org.jivesoftware.smackx.ServiceDiscoveryManager.discoverInfo(ServiceDiscoveryManager.java:512)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.performServiceDiscovery(DiscoveryManager.java:442)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.queryFeatureSupport(DiscoveryManager.java:379)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager.access$3(DiscoveryManager.java:346)
	at de.fu_berlin.inf.dpp.net.discoverymanager.DiscoveryManager$2.run(DiscoveryManager.java:325)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
DEBUG 2014-03-30 15:55:41,667 (DiscoveryManager.java:101) clearing cache entry of contact gabrielwilde@gmail.com/MessagingAe01cc205 but cache entry is empty (a discovery is still running or the last one failed)
ERROR 2014-03-30 15:55:41,667 (SarosNet.java:95) XMPP connection error: 
java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
ERROR 2014-03-30 15:55:41,917 (SarosNet.java:385)  new connection state == ERROR
java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at sun.security.ssl.InputRecord.readFully(Unknown Source)
	at sun.security.ssl.InputRecord.read(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
	at sun.security.ssl.SSLSocketImpl.readDataRecord(Unknown Source)
	at sun.security.ssl.AppInputStream.read(Unknown Source)
	at sun.nio.cs.StreamDecoder.readBytes(Unknown Source)
	at sun.nio.cs.StreamDecoder.implRead(Unknown Source)
	at sun.nio.cs.StreamDecoder.read(Unknown Source)
	at java.io.InputStreamReader.read(Unknown Source)
	at java.io.BufferedReader.fill(Unknown Source)
	at java.io.BufferedReader.read1(Unknown Source)
	at java.io.BufferedReader.read(Unknown Source)
	at org.xmlpull.mxp1.MXParser.fillBuf(MXParser.java:2992)
	at org.xmlpull.mxp1.MXParser.more(MXParser.java:3046)
	at org.xmlpull.mxp1.MXParser.nextImpl(MXParser.java:1144)
	at org.xmlpull.mxp1.MXParser.next(MXParser.java:1093)
	at org.jivesoftware.smack.PacketReader.parsePackets(PacketReader.java:316)
	at org.jivesoftware.smack.PacketReader.access$000(PacketReader.java:46)
	at org.jivesoftware.smack.PacketReader$1.run(PacketReader.java:72)
DEBUG 2014-03-30 15:55:41,932 (SarosNet.java:383)  new connection state == NOT_CONNECTED
INFO  2014-03-31 17:13:20,188 (XMPPAccountStore.java:161) saved 1 accounts
INFO  2014-03-31 17:13:26,589 (XMPPAccountStore.java:161) saved 1 accounts
INFO  2014-03-31 17:13:26,589 (ChangeXMPPAccountAction.java:192) User clicked too fast, running already a connect or disconnect.
INFO  2014-03-31 17:13:27,136 (XMPPAccountStore.java:161) saved 1 accounts
INFO  2014-03-31 17:13:31,484 (XMPPAccountStore.java:161) saved 1 accounts
INFO  2014-03-31 17:13:31,484 (ChangeXMPPAccountAction.java:192) User clicked too fast, running already a connect or disconnect.
INFO  2014-03-31 17:13:35,234 (XMPPAccountStore.java:161) saved 1 accounts
DEBUG 2014-03-31 17:13:38,359 (SarosNet.java:423) started Socks5 proxy on port: 7777 [listening on all interfaces]
DEBUG 2014-03-31 17:13:38,812 (SarosNet.java:433) found interface addresses: [192.168.56.1, 141.22.64.13, fe80:0:0:0:6916:72e8:fdbf:3bcd%wlan0, fe80:0:0:0:6162:f90:6766:5971%eth5, 2002:8d16:400d:0:0:0:8d16:400d]
DEBUG 2014-03-31 17:13:38,812 (SarosNet.java:383)  new connection state == CONNECTING
WARN  2014-03-31 17:13:49,343 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-03-31 17:13:49,343 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.64.13:0
ERROR 2014-03-31 17:13:59,910 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
DEBUG 2014-03-31 17:13:59,925 (SarosNet.java:383)  new connection state == NOT_CONNECTED
DEBUG 2014-03-31 17:13:59,925 (SarosNet.java:423) started Socks5 proxy on port: 7777 [listening on all interfaces]
DEBUG 2014-03-31 17:14:00,331 (SarosNet.java:433) found interface addresses: [192.168.56.1, 141.22.64.13, fe80:0:0:0:6916:72e8:fdbf:3bcd%wlan0, fe80:0:0:0:6162:f90:6766:5971%eth5, 2002:8d16:400d:0:0:0:8d16:400d]
DEBUG 2014-03-31 17:14:00,331 (SarosNet.java:383)  new connection state == CONNECTING
WARN  2014-03-31 17:14:10,726 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /141.22.64.13:0
WARN  2014-03-31 17:14:10,726 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/132.177.123.6:3478 at local address /192.168.56.1:0
WARN  2014-03-31 17:14:19,292 (NullSarosSession.java:64) unexpected method call
WARN  2014-03-31 17:14:19,308 (NullSarosSession.java:64) unexpected method call
ERROR 2014-03-31 17:14:21,370 (SarosNet.java:385)  new connection state == ERROR
googlemail.com:5222 Exception: XMPPError connecting to googlemail.com:5222.; : remote-server-error(502)
	at org.jivesoftware.smack.XMPPConnection.connectUsingConfiguration(XMPPConnection.java:600)
	at org.jivesoftware.smack.XMPPConnection.connect(XMPPConnection.java:1022)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:250)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-31 17:14:21,370 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@2608e8d, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:273)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
DEBUG 2014-03-31 17:14:21,386 (SarosNet.java:383)  new connection state == NOT_CONNECTED
WARN  2014-03-31 17:14:21,386 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@1b993911, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:275)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2014-03-31 17:14:21,402 (SWTUtils.java:193) could not execute runnable de.fu_berlin.inf.dpp.ui.util.SWTUtils$1@1a507b20, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTSync(SWTUtils.java:194)
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSWTSync(SWTUtils.java:157)
	at de.fu_berlin.inf.dpp.ui.util.DialogUtils.popUpYesNoQuestion(DialogUtils.java:242)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:675)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
DEBUG 2014-03-31 17:14:21,402 (SarosNet.java:423) started Socks5 proxy on port: 7777 [listening on all interfaces]
DEBUG 2014-03-31 17:14:21,402 (UPnPServiceImpl.java:295) deleting existing port mappings
ERROR 2014-03-31 17:14:21,417 (DialogUtils.java:250) An internal error ocurred while trying to open the question dialog.
DEBUG 2014-03-31 17:14:21,948 (SarosNet.java:433) found interface addresses: [192.168.56.1, 141.22.64.13, fe80:0:0:0:6916:72e8:fdbf:3bcd%wlan0, fe80:0:0:0:6162:f90:6766:5971%eth5, 2002:8d16:400d:0:0:0:8d16:400d]
DEBUG 2014-03-31 17:14:21,948 (SarosNet.java:383)  new connection state == CONNECTING
WARN  2014-03-31 17:14:21,948 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@373fd1d8, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:248)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
